test/integration/graql/reasoner/resources/recursion/nguyen.gql

```diff
@@ -10,23 +10,15 @@ a-entity sub entity2;
 b-entity sub entity2;
 S sub entity2;
 
-R-rA sub role;
-R-rB sub role;
 R sub relation, relates R-rA, relates R-rB;
```

Comment:
My PR takes this change to the next level, by fully deleting the file!

Reply:
ok will remove this to avoid conflicts